src/api/java/com/minecolonies/api/colony/buildings/IBuildingCanBeHiredFrom.java

```diff
@@ -0,0 +1,7 @@
+package com.minecolonies.api.colony.buildings;
+/**allows citizens from 'training' buildings to be added to hiring lists when implemented by the building.
```

Comment:
Allows upper case and on the new line then. Check out some other classes for the style we're using in class based jadoc